.github/workflows/release.yml

```diff
@@ -1,6 +1,12 @@
 name: goreleaser
 
 on:
+  workflow_dispatch:
+    inputs:
+      dry-run:
+        description: Dry run without publishing
+        type: boolean
+        default: true
```

Comment:
since it's defaulted to true and it uses `push` events, will `dry-run` be always `true` when a tag push?

Reply:
when a tag is pushed the value of `dry-run` should be `null` or empty.